### PR TITLE
refactor(PushV2): add Event Hubs and service reminder leads

### PR DESCRIPTION
**Pull request by LuisFros**
[sc-113075][skip-sc]

### CODE REVIEW

File: push-v2.md [hunk 1]
```diff
@@ -1088,6 +1088,26 @@ Example:
...
+Example:
+
+``` json
+[
+    {
+        "type": "service_reminder_lead",
+        "id": 12095712,
+        "vehicleId": 59821,
+        "time": "2022-01-12T07:06:45",
+        "value": "2022-05-19T18:31:03"
```
Comment (mex) [inline push-v2.md:1107]: The example above doesn't match the example down here. Also why does this have the time part? Shouldn't it just be a date?

Comment (LuisFros) [reply push-v2.md]: Good point. I will update

Comment (mex) [reply push-v2.md]: Couldn't we just use `2022-05-19` instead of `2022-05-19T00:00:00.000Z`?

Comment (mex) [reply push-v2.md]: Alright, then we'll just stick with that.

**Review by mex — APPROVED** [review]

**Pull request merged**